### PR TITLE
Update file structure to include "private node" section

### PR DESCRIPTION
**Pull request by onmax**
Apart from the new section I also updated the content of some pages to improve navigation when the user is reading the content. 

Preview: https://deploy-preview-12--developer-center.netlify.app/build/

### CODE REVIEW

File: build/web-client-rpc.md [hunk 1]
```diff
@@ -52,3 +52,17 @@ For developers, tech-savvies, and validators looking for more advanced control o
 - Verify transactions
 - List all available accounts and their balances
 - Retrieve ZKP state information
+
+
+## Start building now
+
+Now, that you know the differences, why not start building?
```
Comment (sisou) [inline build/web-client-rpc.md]:
```suggestion
Now that you know the differences, why not start building?
```

File: build/index.md [hunk 2]
```diff
@@ -17,9 +17,14 @@ links:
...
     tags: [Validator]
     href: becoming-a-validator
+  - label: Clients
+    title: Web vs RPC
+    tags: ['Web Client', 'Private Node']
+    href: web-client-rpc/
   - label: RPC
-    title: Connect via RPC
-    href: rpc-docs/
+    title: Use the RPC
```
Comment (sisou) [inline build/index.md]:
```suggestion
    title: Use RPC
```

File: build/private-node/index.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+aside: false
+footer: false
+---
+
+<HeadsUp />
+
+# Set up your Nimiq Albatross node
+
+
+Full Control and Advanced Customization Beyond the Web Client.{.subline}
```
Comment (sisou) [inline build/private-node/index.md]:
```suggestion
# Set up your Nimiq Albatross node

Full Control and Advanced Customization Beyond the Web Client.{.subline}
```

**Comment by onmax** [comment]
Feel free to change any of the text or content as you see fit. If you don't like any of the changes, we can always undo them without merging and think of something else. Also, if you think we should include more people like Basti in this discussion, just include them

I'd like to merge this pull request into #10 first, and then move it into the main branch.

File: build/web-client-rpc.md [hunk 4]
```diff
@@ -52,3 +52,17 @@ For developers, tech-savvies, and validators looking for more advanced control o
...
+
+<div flex="~ gap-16 items-center" class="raw" mt-16>
+  <a href="/build/web-client/index.md" arrow pill-sm pill-blue>Web Client</a>
+  <a href="/build/private-node/index.md" arrow pill-sm pill-secondary>Your private node</a>
+</div>
+
+## Learn about the protocol
+
+Curious about how the Nimiq blockchain works? Dive into the [Nimiq Albatross](/protocol/albatross.md) to learn more about the consensus algorithm, block structure, network protocol and more.
```
Comment (sisou) [inline build/web-client-rpc.md]:
```suggestion
Curious about how the Nimiq blockchain works? Dive into the [Nimiq Albatross protocol documentation](/protocol/albatross.md) to learn more about the consensus algorithm, block structure, network protocol and more.
```

File: build/web-client/getting-started.md [hunk 5]
```diff
@@ -36,6 +36,11 @@ client.addHeadChangedListener((head) => {
 })
 ```
 
+
+## Using your private node
+
+Although the Web Client already offers a lot of functionality, you can also set up your own private node for full control and advanced customisation. [Find out more](../web-client-rpc.md).
```
Comment (sisou) [inline build/web-client/getting-started.md]:
```suggestion
Although the Web Client already offers a lot of functionality, you can also set up your own private node for full control and advanced customisation. [Find out more about their differences](../web-client-rpc.md).
```

File: build/private-node/index.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+# Set up your Nimiq Albatross node
+
+Full Control and Advanced Customization Beyond the Web Client.{.subline}
+
+<Tags :tags="$frontmatter.links.map(l => l.tags).filter(Boolean).flat()" mt-24 />
+<Grid  :items="$frontmatter.links" mt-64 />
+
+## Not sure which client to use? {.mt-64}
+
+For many applications it is not necessary to use a private node. Learn more about the differences between the [Web Client and RPC](/build/web-client-rpc.md).{.mt-16}
```
Comment (ines-toupeira) [inline build/private-node/index.md]: Most applications do not require a private node. To learn more, explore the differences between the [Web Client and RPC](/build/web-client-rpc.md).

**Pull request merged**